R/model_averaging.R

```diff
@@ -0,0 +1,97 @@
+#' Compute model weights according to measure
+#' @description Computes model weights according to measure
+#' @param fits List of model fits (return of getModelFits()-function)
+#' @param measure string with type of measure
+#' 
+#' @return Numeric vector of model weights
+#' @export
+get_model_weights <- function(fits, measure = "Loo"){
+  
+  if(measure == "Loo"){
+    preWeights <- fits$Loo
+    measures <- sapply(preWeights, function(x) x$estimates[rownames(x$estimates) == "looic"][1])
+  }
+  if(measure == "WAIC"){
+    preWeights <- fits$WAIC
+    measures <- sapply(preWeights, function(x) x$estimates[rownames(x$estimates) == "waic"][1])
+  }
+  
+  if(measure == "AIC"){
+    measures <- fits$AIC
+  }
+  if(measure == "AICc"){
+    measures <- fits$AICc
+  }
+  if(measure == "logLik"){
+    measures <- -2 * fits$logLik
+  }
+  if(measure == "BIC"){
+    measures <- fits$BIC
+  }
+  
```

Comment:
Could you add a check if `measures` is none of the above issues, and add an error if so with a note to update the function `get_model_weights` with potential new cases?

Reply:
Is done with next commit